VAMobile/src/screens/PaymentsScreen/PaymentHistory/PaymentHistoryScreen.tsx

```diff
@@ -56,26 +48,40 @@ function PaymentHistoryScreen({ navigation }: PaymentHistoryScreenProps) {
   }
 
   const getPickerOptions = useCallback((): Array<yearsDatePickerOption> => {
-    return map(availableYears, (item) => {
-      const year = item.toString()
-      return {
-        label: year,
-        value: year,
-        a11yLabel: year,
-      }
-    })
-  }, [availableYears])
+    if (payments?.meta.availableYears) {
+      return map(payments.meta.availableYears, (item) => {
+        const year = item.toString()
+        return {
+          label: year,
+          value: year,
+          a11yLabel: year,
+        }
+      })
+    } else {
+      return []
+    }
```

Comment:
Small nit, but instead of wrapping the return in the else statement, you can remove the `else` altogether and just return the empty array under the `if` statement

Reply:
updated